src/Audit.NET.PostgreSql/Providers/PostgreSqlDataProvider.cs

```diff
@@ -248,7 +248,7 @@ public IEnumerable<AuditEvent> EnumerateEvents(string whereExpression)
         public IEnumerable<T> EnumerateEvents<T>(string whereExpression) where T : AuditEvent
         {
             var where = string.IsNullOrWhiteSpace(whereExpression) ? "" : $"WHERE {whereExpression}";
-            var sql = $@"SELECT ""{GetDataColumnName(null)}"" FROM {GetFullTableName(null)}"" {where}";
+            var sql = $@"SELECT ""{GetDataColumnName(null)}"" FROM {GetFullTableName(null)} {where}";
             return EnumerateEventsByFullSql<T>(sql);
         }
```

Comment:
The 
 
    GenerateFullTableName(null)

produces

    "myschema"."MyAuditEventTable"

(it is already double-quote delimited)

Thus the extra double quote (before the WHERE) is unnecessary and causing a bug.

Reply:
Thanks! I will add some unit tests and fix the same for the other overload